### PR TITLE
Removes function of unused items, cleans up some existing ones

### PR DESCRIPTION
**Pull request by DominikPanic**
Gifts are not somewhat decent code, yay?

### CODE REVIEW

File: code/game/objects/items/radio/electropack.dm [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
-			if(istype(master.loc, /mob))
-				attack_self(master.loc)
-			else
-				for(var/mob/M in viewers(1, master))
-					if(M.client)
-						attack_self(M)
-	else
-		usr << browse(null, "window=radio")
+	var/mob/living/carbon/C = usr
+	if(C.stat != CONSCIOUS || C.restrained() || C.back == src)
```
Comment (Rohesie) [inline code/game/objects/items/radio/electropack.dm]:
```suggestion
	if(C.incapacitated() || C.back == src)
```

File: code/game/objects/items/violin.dm [hunk 2]
```diff
@@ -1,393 +1,7 @@
...
-	var/max_lines = 50
-
-/obj/item/violin/proc/playnote(var/note as text)
-	//to_chat(world, "Note: [note]")
-	var/soundfile
-	/*BYOND loads resource files at compile time if they are ''. This means you can't really manipulate them dynamically.
-	Tried doing it dynamically at first but its more trouble than its worth. Would have saved many lines tho.*/
-	switch(note)
-		if("Cn1")	soundfile = 'sound/violin/Cn1.mid'
-		if("C#1")	soundfile = 'sound/violin/C#1.mid'
```
Comment (Rohesie) [inline code/game/objects/items/violin.dm]: There's a ton of sounds here that could be removed from the code if we are removing the violin's functionality.

Comment (spookydonut) [reply code/game/objects/items/violin.dm]: you cant cut back on violins you'll regret this

Comment (DominikPanic) [reply code/game/objects/items/violin.dm]: The sounds already don't exist

Comment (Rohesie) [reply code/game/objects/items/violin.dm]: You've made an enemy for life.

**Comment by Rohesie** [comment]
Think you can remove the sounds now that their references are gone?
After that, this looks ready.

**Pull request merged**